netsim/ansible/templates/initial/cumulus.j2

```diff
@@ -62,7 +62,7 @@ service lldpd restart
 #
 # Enable FRR modules
 #
-{% set modlist = {'bgp':'bgpd','ospf':['ospfd','ospf6d'],'isis':'isisd','vrf':'bgpd'} %}
+{% set modlist = {'bgp':'bgpd','ospf':['ospfd','ospf6d'],'isis':'isisd','vrf':'bgpd','gateway':'vrrpd'} %}
```

Comment:
There is another instance in ```netsim/templates/provider/clab/frr/daemons.j2``` that needs to be updated like this

Reply:
True, but VRRP needs a control- and data-plane components (macvlan interface). It was pretty simple to add data plane component to Cumulus Linux with **ifupdown2**, not sure how much work it would be to add it to FRR the way it's currently implemented (and I'm not about to go exploring).

But yeah, once that's in place the list of FRR modules will have to be updated.